src/style.css

```
@@ -1,3 +1,76 @@
+/* stylelint-disable selector-class-pattern */
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  --main-blue-color:#00acdc;
+}
+
+.img {
+  padding-left: 50px;
+  align-self: center;
+}
+
+header {
```

Use only class selector
```suggestion
.header {
```